### ISSUE TITLE
Electrum is inaccessible to visually impaired users

### ISSUE THREAD

**Issue by meZee** [spesmilo/electrum#1036]
Hello, a buddy of mine is visually impaired and isn't local to me, and he reports that his screen reader(NVDA) isn't unable to read visual elements in Electrum aloud, it has no issue with the bottom row of buttons, but the menu, window panes, and children elements aren't recognized.

**Comment by rgenito** [comment]
interesting, and he is able to use other bitcoin wallets without this problem?

**Comment by Stemby** [comment]
I found this bug report in Debian BTS, with some suggestions:

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=734235

I hope this can be useful.

Ciao!

**Comment by ot4me** [comment]
@rgenito as far as I remember comments from other visually impaired user on reddit no, none of the wallets were any good.

**Comment by ecdsa** [comment]
would it be fixed by adding more Tooltips ?

**Comment by pzajda** [comment]
@rgenito I am blind using NVDA, and I can use Bitcoin Core without major issue.

@Stemby NVDA is a screen reader for windows, so the solution to install a package won't be applicable unless there is a port for Windows which could enhance accessibility.
For the CLI interface, is it really reasonable to ask a beginer for example to use CLI when there is an GUI which will do the job easyly? Because finally, the Debian recommandation for blind users is to use a harder way to be able to use Electrum easyer. I tried to convince some blind users to use CLI for others softwares, but unsuccessfully...

@ecdsa yes, it could make the accessibility better for blind users with tooltips, if these tooltips can be read by screen readers.
I know QT is used for the GUI, is it possible to enable accessibility for QT in all the project when used with Python? I know QT has accessibility plugins, but unfortunatly not enough to give you more precisions.

**Comment by pzajda** [comment]
It seems PYQT4 doesn't implement accessibility at all.
It know QT5 implement it by default, maybe Python binding also implement accessibility?
Thanks.

**Comment by bauerj** [comment]
Hey @pzajda, we're using PyQt5 now so I hope accessibility has improved since. Could you please check if it's any better now?

**Comment by bauerj** [comment]
I'm closing this issue now since we don't know if the issue still persists. If someone with a screenreader or experience with accessibility can provide more insight, please reopen.

We would be happy to improve the situation but we need some pointers like what needs to be improved.

**Comment by ecdsa** [comment]
not directly related, but it would be nice to be able to trigger the dark color scheme with a command line option, e.g. "electrum --dark"
